src/views/LoginView.vue

```diff
@@ -3,17 +3,13 @@
     <!--//////////////////////////
      /////Welcome Section /////
     //////////////////////////-->
-    <section v-if="!isLoggingIn">
+    <section>
       <v-container fluid id="welcome-container" class="welcomeContainer">
         <nav class="d-flex justify-space-between align-center mb-2">
-          <img class="cardLogo" src="/img/loginView/logo-cards-behind.svg">
-          <v-btn
-            variant="text"
-            class="text-h6"
-            @click ="$refs.usernameInput.focus()"
-          >
+          <img class="cardLogo" src="/img/loginView/logo-cards-behind.svg" />
+          <v-btn variant="text" class="text-h6" @click="directToSignup">
```

Comment:
```suggestion
          <v-btn variant="text" class="text-h6" @click="scrollAndFocusLogin(true)">
```

Reply:
Instead of changing the page to signup let's make this button say login or signup depending on the route and just scroll and focus without changing the page